packages/mesos/extra/windows/dcos-mesos-slave-public.xml

```diff
@@ -0,0 +1,18 @@
+<configuration>
+  <id>dcos-mesos-slave</id>
+  <name>DCOS Mesos Windows Slave</name>
+  <description>Windows Service for the DCOS Mesos Agent</description>
+  <executable>C:\DCOS\mesos\bin\mesos-agent.exe</executable>
+  <arguments>--master="zk://192.168.255.5:2181/mesos" --work_dir="C:\DCOS\mesos\work" --runtime_dir="C:\DCOS\mesos\work" --launcher_dir="C:\DCOS\mesos\bin" --external_log_file="C:\DCOS\mesos\log\mesos-service.err.log" --log_dir="C:\DCOS\mesos\log" --ip="10.0.0.4" --isolation="windows/cpu,filesystem/windows" --containerizers="docker,mesos" --attributes="os:Windows;public_ip:yes" --default_role="slave_public"</arguments>
```

Comment:
these are the current files we use on windows. We have not got to the configuration part of the porting because this is a work in progress and we will do that in due course. Our goal was to use the build we have produced from pkgpanda to continue installing in our current way as a stop-gap. The more we tear out of this the more we cannot actually use pkgpanda in our deployment.
I will stop that from happening and remove these files for you though.